test/mm_service.wsdl

```diff
@@ -8,20 +8,63 @@
                   name="service"
                   targetNamespace="http://example.com/service">
   <wsdl:types>
-    <xsd:schema>
+    <xsd:schema targetNamespace="http://example.com/service">
       <xsd:import namespace="http://example.com/importer"
                   schemaLocation="importer.xsd"/>
+      <xsd:element name="getPosers">
+        <xsd:complexType>
+          <xsd:sequence/>
+        </xsd:complexType>
+      </xsd:element>
+      <xsd:element name="getPosersResponse">
+        <xsd:complexType>
+          <xsd:sequence>
+            <xsd:element name="fluster" type="FlusterType"/>
+          </xsd:sequence>
+        </xsd:complexType>
+      </xsd:element>
+      <xsd:simpleType name="FlusterType">
+        <xsd:annotation>
+          <xsd:documentation>This pattern triggered a bug in wh.erl</xsd:documentation>
+        </xsd:annotation>
+        <xsd:restriction base="xs:string">
+          <xsd:pattern value="[\w\d\p{Zs}\-/_.,:]*"/>
+          <xsd:minLength value="1"/>
+          <xsd:maxLength value="80"/>
+        </xsd:restriction>
+      </xsd:simpleType>
```

Comment:
I tried reverting the fix for this and test started failing, so this tests the right thing.